requirements.txt

```diff
@@ -1,2 +1,3 @@
+pytest
```

Comment:
Nitpick (based on the same mistake I made), but `git add -p` would have prevented this from being added to this commit. You can rebase this commit and the commit removing this line and it *should* (if I understood this correctly) cancel out this change.